content/en/about-the-docs/_index.md

```diff
@@ -40,25 +34,29 @@ Definitions:
 * **End of security**: This release will no longer be eligible for security patches after the date shown.
 * **End of support**: Support (including documentation) provided by R3 is no longer available after this date.
 
-| **Version** | **Date of Release** | **End of Maintenance** | **End of Security**   | **End of Support**    |
-|:-------------:|:-------------------:|:----------------------:|:---------------------:|:---------------------:|
-| **Corda 5.0** | Jul-23              | Jan-24    | Jan-24   | Jan-24   |
-| **Corda 5.1** | TBC                 | TBC    | TBC   | TBC   |
+{{< note >}}
+All dates refer to the end of the month indicated.
+{{< /note >}}
 
+### Corda 5
 
-## Reporting an Error in the Docs
-In 2023, the R3 Technical Writing team will be making big changes to our documentation and our overall content strategy. As we make improvements, we’d love to get your feedback on what we’re doing well, and what we could be doing better. 
+{{< snippet "corda-5.0/end-of-life-corda5.md" >}}
 
-Click [here](mailto:docs@r3.com) to email the R3 Technical Writing team. To help us deal with your feedback effectively, please include all of the following:
+### Corda 4
 
-* Your name
-* Your company name
-* Your email address (if different from the one being used to report)
-* The URL of the documentation page (if it is an issue being reported)
+{{< snippet "corda-4/end-of-life-corda4.md" >}}
 
-Please include as much detail as you can in your feedback, as it is valuable to us and it will be read and shared with the rest of the team. A ticket is then created for any issues received, which will be quickly triaged and if necessary, assigned to a Technical Writer to effect changes.
+## Documenting Corda 5
+The Corda 5 documentation is structured differently than in previous versions of Corda. Previously, Corda Enterprise and Corda Community content was published in separate documentation sets. 
+From Corda 5.0 onwards, there is one documentation set, but the features that relate to Corda Enterprise are marked to avoid confusion. 
```

Comment:
```suggestion
From Corda 5.0 onwards, there is one documentation set, but the features that relate only to Corda Enterprise are marked to avoid confusion. 
```